client-ui/src/app/components/lineage/lineage-graph/lineage-graph.component.ts

```diff
@@ -87,10 +74,28 @@ export class LineageGraphComponent implements OnInit, OnChanges, AfterViewInit {
         this.selectedNodeChange.emit(nodeId)
       })
     })
+    this.refreshGraph()
     this.refreshSelectedNode()
     this.refreshAttributeGraph()
   }
 
+  public refreshGraph(): void {
```

Comment:
Why it should be public?

Reply:
a typo. This used to be a `ngAfterViewInit` before.
Thanks for pointing out.
Fixed.